### DISCUSSION TITLE
htpasswd file management capability

### DISCUSSION THREAD

**Discussion by Rabbit10** [Kozea/Radicale#1710] [Ideas]
Bonjour,
est-ce qui serait possible d'ajouter quelque chose pour gérer les utilisateurs ( htpasswd ) directement sur la page web de radicale ? 
il y a [php-apache2-basic-auth-manager](https://github.com/rafaelgou/php-apache2-basic-auth-manager) mais ça serais plus propre si un système de ce genre existerai en natif

**Comment by Unrud** [comment]
I think that account management is out of scope for Radicale. If we also forward POST requests to web plugins, I guess it could be implemented as a plugin.

**Comment by Rabbit10** [comment]
ok pas de problème, je suis incapable de créer ce genre de chose, je mettrais en place [php-apache2-basic-auth-manager](https://github.com/rafaelgou/php-apache2-basic-auth-manager) en attendant qu'un plugin de ce genre sorte. merci
